src/services/milestones/getApprovedKeys.test.js

```diff
@@ -0,0 +1,124 @@
+const { assert, expect } = require('chai');
```

Comment:
I found some functions that can write unit test for them,  I wrote some unit tests for this function